ml-agents/mlagents/trainers/settings.py

```diff
@@ -840,6 +859,8 @@ def from_argparse(args: argparse.Namespace) -> "RunOptions":
             "torch_settings": {},
         }
         if config_path is not None:
+            # If we're loading for a file, make sure we have strict on
+            configured_dict["strict"] = True
```

Comment:
Would it be simpler to remove these two lines and make strict True by default rather than False line 817 ?